src/harness/os/linux.c

```diff
@@ -301,3 +301,11 @@ size_t OS_ConsoleReadPassword(char* pBuffer, size_t pBufferLen) {
     tcsetattr(STDIN_FILENO, TCSANOW, &old);
     return len;
 }
+
+char* OS_Dirname(char* path) {
+    return dirname(path);
+}
+
+char* OS_Basename(char* path) {
+    return basename(path);
+}
```

Comment:
These functions may modify their argument.
Perhaps also use a static buffer, as it does on Windows?